bin/weewx/drivers/vantage.py

```diff
@@ -1481,6 +1502,14 @@ def _archive_datetime(datestamp, timestamp):
                       0, 0, -1)                     # have OS guess DST
         # Convert to epoch time:
         ts = int(time.mktime(time_tuple))
+        # fix for hour mismatches at begin of summertime
+        # calculate a time tuple from the UTC time stamp
+        utc_time_tuple = time.localtime(ts)
```

Comment:
Tom, you stated: "nothing should be thrown away". I agree. The current situation is: the user notices weewx is 'storing nothing anymore to the weewx database since summert time change'. At last he/she decides to clear the logger memory. As by 'magic' weewx is working again! Hurray!
How many weewx records are lost in this process do you think? (Please don't answer!).